iOS/DuckDuckGo/SuggestionTrayViewController.swift

```diff
@@ -186,7 +186,7 @@ class SuggestionTrayViewController: UIViewController {
             variableHeightConstraint.constant = Constant.suggestionTrayInitialHeight
         }
 
-        variableWidthConstraint.constant = width
+        variableWidthConstraint.constant = width + 32
```

Comment:
I think the width should be changed in calling site instead, so in `applyWidthToTrayController`. Here, the value is just applied, and it's the MainVC which is expecting the tray to be wider.